playbooks/roles/postgresql_aggregation/tasks/main.yml

```diff
@@ -5,14 +5,14 @@
     name: "{{ item }}"
     install_recommends: yes
     state: present
-  with_items: postgresql_aggregation_packages
+  with_items: "{{ postgresql_aggregation_packages }}"
   tags: ['postgresql_aggregation']
 
 - name: install requirements for role
   pip:
     name: "{{ item }}"
     state: present
-  with_items: postgresql_aggregation_requirements
+  with_items: "{{ postgresql_aggregation_packages }}"
```

Comment:
Hey @melvinsoft just a quick fix here i think items should be "{{ postgresql_aggregation_requirements }}" . thanks